sdk/core/Azure.Core/src/RequestContent.cs

```diff
@@ -15,6 +17,9 @@ namespace Azure.Core
     /// </summary>
     public abstract class RequestContent : IDisposable
     {
+        private static readonly Encoding s_UTF8NoBomEncoding = new UTF8Encoding(false);
+        private static readonly ObjectSerializer s_DefaultSerializer = new JsonObjectSerializer();
```

Comment:
We should add a static like that to JsonObjectSerializer. Otherwise we will end up with many of such statics all over Azure.Core.

Reply:
Done.